tests/test_tuplists.py

```diff
@@ -192,10 +192,21 @@ def fmt(_tup):
         self.assertEqual(a, self.prop1)
 
     def test_to_dictlist(self):
-        self.assertEqual(self.prop1.to_dictlist([1, 2, 3, 4]), self.prop2)
+        # self.assertEqual(self.prop1.to_dictlist([1, 2, 3, 4]), self.prop2)
```

Comment:
why comment this line?

Reply:
it is a mistake, I had commented it to try something and forgot to uncomment